core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -15595,15 +15585,15 @@ where exists(select * from dept where emp.deptno = dept.deptno)]]>
 LogicalProject(EMPNO=[$0], ENAME=[$1], JOB=[$2], MGR=[$3], HIREDATE=[$4], SAL=[$5], COMM=[$6], DEPTNO=[$7], SLACKER=[$8])
   LogicalJoin(condition=[=($7, $9)], joinType=[inner])
     LogicalTableScan(table=[[CATALOG, SALES, EMP]])
-    LogicalProject(DEPTNO=[$0])
+    LogicalProject(DEPTNO3=[$0])
```

Comment:
this suggests that an ALIAS is missing somewhere

Reply:
I have rolled back the relevant changes, it seems to be caused by the logic above.